examples/rvc-app/rvc-common/include/rvc-device.h

```diff
@@ -89,6 +96,14 @@ class RvcDevice
      */
     void HandleOpStateGoHomeCallback(Clusters::OperationalState::GenericOperationalError & err);
 
+    /**
+     * @brief Handles the check to see if the Selected Locations can be changed
+     * @param locationText if setting the selected location failed, a description of the mode or condition prohibiting the change
+     *        size kMaxSizeStatusText + 1 for termination char
+     * @return true if Selected Locations can be changed
+     */
+    bool HandleIsSetSelectedLocationCallback(char * locationText);
```

Comment:
Correct. This is not being used. It's something that is going to be added later on to make the example more realistic by having it interact with the other RVC clusters. This will be removed.